BREAKING.md

```diff
@@ -50,6 +51,12 @@ This section details the desktop browser, JavaScript framework, and mobile platf
 
 <h2 id="version-7x-components">Components</h2>
 
+<h4 id="version-7x-input">Input</h4>
+
+`ionChange` will no longer be emitted when the `value` of `ion-input` is modified externally. `ionChange` will only be emitted from user committed changes, such as typing in the input and the input losing focus or from clicking the clear action within the input.
```

Comment:
```suggestion
`ionChange` is no longer emitted when the `value` of `ion-input` is modified externally. `ionChange` is only emitted from user committed changes, such as typing in the input and the input losing focus or from clicking the clear action within the input.
```

Minor nit, but we use present tense in the rest of the breaking changes guide for v7.